includes/functions/functions_general.php

```diff
@@ -1674,7 +1674,9 @@ function zen_js_zone_list($country, $form, $field, $showTextField = true) {
       $num_state = 1;
       while (!$states->EOF) {
         if ($num_state == '1') $output_string .= '    ' . $form . '.' . $field . '.options[0] = new Option("' . PLEASE_SELECT . '", "");' . "\n";
-        $output_string .= '    ' . $form . '.' . $field . '.options[' . $num_state . '] = new Option("' . $states->fields['zone_name'] . '", "' . $states->fields['zone_id'] . '");' . "\n";
+        $output_string .= '    ' . $form . '.' . $field . '.options[' . $num_state . '] = new Option("' . $states->fields['zone_name'] . '", "' . $states->fields['zone_id'] . '"';
+        if($states->fields['zone_id'] == $zone_id) $output_string .= ',1,1';
```

Comment:
What does the `,1,1` refer to here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement/Option

It makes the option being added selected.